.ahoy.yml

```diff
@@ -48,7 +48,7 @@ commands:
 
   drush:
     usage: Run drush commands in cli container.
-    cmd: docker-compose exec -T cli drush "$@"
+    cmd: docker-compose exec -e COLUMNS=120 -T cli drush "$@"
```

Comment:
does the $COLUMNS env var not pass nicely through ahoy here to avoid hardcoding a value? $COLUMNS is a default env var for the current local session (that varies as you resize a window)

Reply:
Right. I didn't realize that that was an actual env var - I thought that it was a placeholder. That does indeed work 👍🏼 

I'll do you one better though. ...just passing plain `-e` without specifying any parameters actually seems to pass the defaults, which has the nice side-effect of allowing coloured output pass-through. Here's a couple of before/after screenshots:

![image](https://user-images.githubusercontent.com/2423362/173486047-4f7bdd93-ffac-45e3-8709-dbb874d7b065.png)

![image](https://user-images.githubusercontent.com/2423362/173486132-57da3363-85ba-416f-b2e8-5e8b9ae95f90.png)